README.md

```diff
@@ -1 +1,38 @@
-# elm-workshop
\ No newline at end of file
+# Codher Purely Functional Interfaces in Elm
+
+## Agenda
+
+- Getting started with Elm and Create Elm App
+- Intro into Functional Programming
+- Rendering HTML and CSS
+- Building interactive applications
+
+## Introduction
+
+Welcome to Codher's Elm Workshop!
+
+Functional Programming is fun and easy if you learn by doing.
+
+Have you ever tried programming in a language, where compiler is your best friend? Imagine an app without errors at run-time.
+
+Sounds promising?
+
+Immutabile, Pure and Reactive by nature all that awailable now!
+
+Get an introduction to Elm Compiler and Time-Travelling Debugger and start coding!
```

Comment:
In this workshop you'll get an introduction to the Elm Compiler (so you can build your first application or something), and the Time-Travelling debugger (so you...) and start coding in Elm!